docs/guides/nrfconnect_examples_cli.md

```diff
@@ -95,109 +95,196 @@ These commands are not available in the standard Zephyr shell.
 They are currently used for testing purposes and allow only to get some
 information about CHIP stack state, but not to modify it.
 
-To enable using CHIP commands, set the `CONFIG_CHIP_ZEPHYR_SHELL=y` Kconfig
+To enable using CHIP commands, set the `CONFIG_CHIP_LIB_SHELL=y` Kconfig
 option in the `prj.conf` file of the given example.
 
-### Listing CHIP-specific commands
+Every invoked command must be preceded by the `matter` prefix.
 
-To list all available CHIP-specific commands, enter `chip` in the command line
-and press the Tab key. This will list the available commands:
+See the following subsections for the description of each CHIP-specific command.
+
+### `onboardingcodes`
+
+Handles group of commands used to view information about device onboarding codes. You can use it without any subcommand to print all available onboarding codes or add a specific subcommand.
 
 ```shell
-uart:~$ chip
-  qrcode             qrcodeurl          setuppincode       discriminator
-  vendorid           productid          manualpairingcode  bleadvertising
-  nfcemulation
+uart:~$ matter onboardingcodes
+QRCode:             MT:W0GU2OTB00KA0648G00
+QRCodeUrl:          https://dhrishi.github.io/connectedhomeip/qrcode.html?data=MT%3AW0GU2OTB00KA0648G00
+ManualPairingCode:  34970112332
 ```
 
-See the following subsections for the description of each CHIP-specific command.
+It can also take the following subcommands presented below.
 
-#### `qrcode`
+#### qrcode
```

Comment:
I think the current hierarchy works well, but indeed we could include the name of the main command in the subcommand header, like `#### (onboardingcodes) qrcode`

Also, shouldn't this `####` for qrcode be a `###`?

Reply:
If it comes to the including command names in subcommands - I don't find it useful, as it's a lack of consequence, because following this pattern we should also add `matter` command. That will result in having listed commands like `matter ble adv start`, what IMO is not looking good, especially that we have examples of usage for each command.

if it comes to using ### instead of ### I did it intentionally to highlight main group command from the subcommands. Is it wrong and all should be on the same level?